scripts/prerequisite_check.sh

```diff
@@ -37,102 +38,149 @@ YELLOW='\033[1;33m'
+            return 0
+        else
+            echo -e "${RED}❌ yq version $version is too old. Please install yq v4+ from https://github.com/mikefarah/yq/${NC}"
+            return 1
+        fi
+    else
+        echo -e "${RED}❌ yq is not installed. Please install yq v4+ from https://github.com/mikefarah/yq/${NC}"
+        return 1
+    fi
+}
 
-  if ! dpkg-query -W -f='${Status}' gcc-multilib 2>/dev/null | grep "ok installed" &>/dev/null; then
+if ! dpkg-query -W -f='${Status}' gcc-multilib 2>/dev/null | grep "ok installed" &>/dev/null; then
     echo "📥 Installing gcc-multilib..."
     sudo apt update
     sudo apt -y install gcc-multilib
-  else
+else
     echo "✔️ gcc-multilib is already installed."
-  fi
+fi
 
-  if ! check_command rustc; then
-    echo -e "${RED}❌ Rust is not installed. Please install it from https://rustup.rs/${NC}"
-    exit 1
-  else
-    echo "✔️ Rust is already installed."
-  fi
-
-  if ! check_command cosign; then
-    echo "📥 Installing Cosign..."
-    curl -O -L "https://github.com/sigstore/cosign/releases/latest/download/cosign-linux-amd64"
-    sudo mv cosign-linux-amd64 /usr/local/bin/cosign
-    sudo chmod +x /usr/local/bin/cosign
-  else
-    echo "✔️ Cosign is already installed."
-  fi
-
-  if ! check_command docker; then
-    echo -e "${RED}❌ Docker is not installed. Please install it from https://docs.docker.com/engine/install/ubuntu/${NC}"
-    exit 1
-  else
-    echo "✔️ Docker is already installed."
-  fi
-
-  missing=()
-  for cmd in kubectl yq sed gh pkg-config jq; do
+missing=()
+for cmd in cosign docker kubectl sed gh pkg-config jq helm git; do
     if ! check_command "$cmd"; then
-      missing+=("$cmd")
+        missing+=("$cmd")
     fi
-  done
+done
+
+if ! check_yq_version; then
+    missing+=("yq(v4+)")
+fi
 
-  if ! dpkg -s libssl-dev &>/dev/null; then
+if ! dpkg -s libssl-dev &>/dev/null; then
     missing+=("libssl-dev")
-  fi
+fi
 
-  if [ ${#missing[@]} -ne 0 ]; then
+if [ ${#missing[@]} -ne 0 ]; then
     echo -e "${RED}❌ Missing required tools/packages:${NC} ${missing[*]}"
     exit 1
-  fi
+fi
 
-  if ! kubectl cluster-info &>/dev/null; then
+if ! kubectl cluster-info &>/dev/null; then
     echo -e "${RED}❌ No Kubernetes cluster detected via kubectl. Is your cluster running?${NC}"
     exit 1
-  fi
-  echo "✅ Installed all external executable"
+fi
+echo "✅ Installed all external executable"
+
 LILAC='\033[1;35m'
 RESET='\033[0m'
 printf "${LILAC}"
 cat <<EOF
 
 
-## Check access to 'scone.cloud' images
+## Check access to repositories
 
-We check that we can pull some SCONE container images that we need to execute
-the transformations. If this fail, please do the following:
-
-- generate an access token following these instructions: <https://sconedocs.github.io/registry/#create-an-access-token>
-
-- 
+We check access to required GitHub repositories and container registry.
 
 EOF
 printf "${RESET}"
 
+echo -e "${YELLOW}🔍 Checking access to GitHub repositories...${NC}"
```

Comment:
We will not give access to the repos - we provide .deb packages to install this software - see script install-sconecli